include/materials/MFEMMaterial.h

```diff
@@ -2,17 +2,22 @@
 
 #include "MFEMGeneralUserObject.h"
 #include "MFEMCoefficient.h"
+#include "PropertyManager.h"
 #include "coefficients.h"
 
 class MFEMMaterial : public MFEMGeneralUserObject
 {
 public:
   static InputParameters validParams();
+  static std::vector<std::string> subdomainsToStrings(std::vector<SubdomainName> blocks);
+  static libMesh::Point pointFromMFEMVector(const mfem::Vector & vec);
 
   MFEMMaterial(const InputParameters & parameters);
   virtual ~MFEMMaterial();
 
-  virtual void storeCoefficients(platypus::Subdomain & subdomain) {}
+  const std::vector<SubdomainName> & getBlocks() const { return _block_ids; }
```

Comment:
Could be worth splitting the methods (and the `_block_ids` property) out into its own class that we then inherit from in a future PR, like `BlockRestrictable` in MOOSE (include/interfaces/BlockRestrictable.h)?

We'll also want to get the list of blocks using similar methods for other objects that may be domain restricted (such as kernels), and this would save having to implement them multiple times. Boundary conditions will have a similar story with boundary attributes (similar to MOOSE's `BoundaryRestrictable`)